### PR TITLE
[C++ API Parity] - TripletMarginLoss

### PR DESCRIPTION
**Pull request by PyExtreme**
Hi @yf225 , I had to create a new branch to tackle merge conflict since I am using cloud due to some limitations on my PC. Therefore, I don't have enough command there.

Also, I have incorporated the changes you have put before here
https://github.com/pytorch/pytorch/pull/27613

Also, it would be great if you could recommend me some resources to work smmothly on GCP..:-D

Thank you

### CODE REVIEW

File: torch/csrc/api/src/nn/modules/loss.cpp [hunk 1]
```diff
@@ -77,5 +77,18 @@ Tensor CosineEmbeddingLossImpl::forward(
   return F::cosine_embedding_loss(input1, input2, target, options);
 }
 
+/ ============================================================================
```
Comment (yf225) [inline torch/csrc/api/src/nn/modules/loss.cpp]: We are missing a `/` here:
```suggestion
// ============================================================================
```

**Comment by yf225** [comment]
@PyExtreme curious what's your current workflow on GCP? Are you able to run `git pull` / `git push` on it?

**Comment by PyExtreme** [comment]
@yf225 , I have made the change. 
I am sorry for these silly mistakes which could have been avoided on careful review. 

So there are two problems, I face while using GCP.

1. I have to use vim/nano for writing code. I am now gonna use codeserver for this but the pre-release which is compatible with the vscode version, seems like does not have a password based mechanism to prevent someone else accessing my IP.

2. Due to above mentioned text editors, merging becomes a bit tough. Are there any good text editors or a source-code editor like VSCode which can help me fix conflicts easily?

Thank you so much for your time. I have gained a lot from here.

**Comment by PyExtreme** [comment]
@yf225 , The tests for TripletMarginLoss are getting skipped while running`
./build/bin/test_api --gtest_filter=ModulesTest* --gtest_stack_trace_depth=10 --gmock_verbose=info`
and 
`./build/bin/test_api --gtest_filter=FunctionalTest* --gtest_stack_trace_depth=10 --gmock_verbose=info`

Is there anything which I am missing? Due, to it, I am not able to identify the mistakes.

Thank you

**Comment by PyExtreme** [comment]
@yf225 , could you please have a look at it. I have made all the necessary changes.

**Review by yf225 — APPROVED** [review]
@PyExtreme Sorry for the delay and thanks so much for the awesome work! For fixing the code conflicts, I think as long as we are able to edit the source code we should be able to find all the `<<<<<` occurrences and then fix them. I usually just use vim / Sublime Text for it :D

**Pull request closed without merging**